src/ansible_runner/display_callback/callback/awx_display.py

```diff
@@ -352,6 +344,18 @@ def __init__(self):
         # NOTE: Ansible doesn't generate a UUID for playbook_on_start so do it for them.
         self.playbook_uuid = str(uuid.uuid4())
 
+    def set_options(self, *args, **kwargs):
+        base_config = C.config.get_configuration_definition(
+            DefaultCallbackModule._load_name, plugin_type='callback'
+        )
+        my_config = C.config.get_configuration_definition(
+            self._load_name, plugin_type='callback'
+        )
+        C.config.initialize_plugin_configuration_definitions(
+            'callback', self._load_name, base_config | my_config
+        )
+        return super().set_options(*args, **kwargs)
```

Comment:
Not related, but since I'm in here making a mess, this fixes the options issue with callbacks not matching the default from a config perspective.